File: app/src/androidTest/java/de/westnordost/streetcomplete/data/osm/persist/test/DisabledTestQuestType.java
package de.westnordost.streetcomplete.data.osm.persist.test;

import de.westnordost.streetcomplete.R;

public class DisabledTestQuestType extends TestQuestType
{
	@Override public int getDefaultDisabledMessage()
	{
		return R.string.default_disabled_msg_go_inside;
	}
}

File: app/src/androidInstrumentedTest/kotlin/de/westnordost/streetcomplete/data/osm/osmquests/DisabledTestQuestType.kt
package de.westnordost.streetcomplete.data.osm.osmquests

import de.westnordost.streetcomplete.resources.Res
import de.westnordost.streetcomplete.resources.default_disabled_msg_go_inside

class DisabledTestQuestType : TestQuestType() {
    override val defaultDisabledMessage = Res.string.default_disabled_msg_go_inside
}
